src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -78,9 +79,10 @@ public void setDashboards(final OpenshiftRequestContext context, String kibanaVe
 
         if (action.v2() != null && !Project.EMPTY.equals(action.v2())) {
             boolean defaultIndexPatternExists = pluginClient.documentExists(context.getKibanaIndex(), INDICIES_TYPE, action.v2().getName());
-            boolean kibanaConfigExists = pluginClient.documentExists(context.getKibanaIndex(), CONFIG_DOC_TYPE, kibanaVersion);
-            if(!defaultIndexPatternExists || !kibanaConfigExists){
+            GetResponse config = pluginClient.getDocument(context.getKibanaIndex(), CONFIG_DOC_TYPE, kibanaVersion);
+            if(!defaultIndexPatternExists || !config.isExists() || StringUtils.isBlank(kibanaUtils.getDefaultIndexPattern(config))){
```

Comment:
can `pluginClient.getDocument` return `null`?

Reply:
It's a wrapper around the ES calls so I don't see how.  If response doesnt find anything, AFAIK it returns an object because it has the exist method.  If you prefer, I could update, for safety to additionally check null'

Reply:
If it can't be `null`, then no need to check